CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 ### Added 
 
 - Add `SitterMainTabView` for the sitter with stub view for tabs - [#108](https://github.com/ios-course/ironfoudation-team-project/pull/108)
+- Add new storage keys and names to store preferences and current sitter information  in `KeyValueStorage` - [#109](https://github.com/ios-course/ironfoudation-team-project/pull/109)
```

Comment:
nit:
```suggestion
- Add new storage keys and names to store preferences and current sitter information in `KeyValueStorage` - [#109](https://github.com/ios-course/ironfoudation-team-project/pull/109)
```

Reply:
nit:

In general, you can make it even simpler:
```suggestion
- Update storage keys and storage names to use with the `KeyValueStorage` - [#109](https://github.com/ios-course/ironfoudation-team-project/pull/109)
```

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/109/commits/a0817ad7ed8484d31255be21fc31c16539bcb076